modules/ossm-cr-example.adoc

```diff
@@ -211,6 +214,13 @@ spec:
         config:
           env: {}
           envSecrets: {}
+        persistence:
+          enabled: true
+          storageClassName: ""
+          accessMode: ReadWriteOnce
+          capacity:
+            requests:
+              storage: 5Gi
```

Comment:
This is accurate, but I don't think `storage` is honored (i.e. users can change this value, but it doesn't get applied correctly; there's an open issue for this).